src/device/mod.rs

```diff
@@ -5,4 +5,5 @@ pub mod console;
 pub mod gpu;
 #[cfg(feature = "alloc")]
 pub mod input;
+#[cfg(feature = "alloc")]
 pub mod net;
```

Comment:
Is it possible to keep a no-alloc subset of the net module?

Reply:
May be another PR, their implementation can be really different.